HACKING.md

```diff
@@ -97,3 +97,15 @@ For example, I've the following in my `.emacs.d/init.el`:
 (add-hook 'before-save-hook 'gofmt-before-save)
 (require 'go-rename)
 ```
+
+## Testing within VSCode
```

Comment:
nice!
I think that's in fact the right place to add it, maybe we should delete the one in `CONTRIBUTING.md` then ?

and use the paragraph for how to launch the vscode extension from the other file